apps/aggregates/src/utils/index.ts

```diff
@@ -22,11 +22,15 @@ export function createPalette(labels: Array, palette: Array): Object {
  * @params descending - If True, the dataset will be sorted in reverse order
  */
 export function sortData(
-  data: Array,
-  by: String,
-  descending: Boolean = false
-): Array {
-  const dataType = [...new Set(data.map((row) => typeof row[by]))];
+  data: Record<string, any>,
+  by: string,
+  descending: boolean = false
+) {
+  const dataType = [
+    ...Array.from(
+      new Set(data.map((row: Record<string, any>) => typeof row[by]))
+    ),
+  ];
```

Comment:
This is a bit hard to read, might split it to a function so the name can give some context e.g. 'getUniqueDataTypes()'.

Reply:
That's a good idea